drivers/metal/SCsub

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+Import("env")
+
+env_metal = env.Clone()
+if '-fno-exceptions' in env_metal['CXXFLAGS']:
+    env_metal['CXXFLAGS'].remove('-fno-exceptions')
```

Comment:
We have to use exceptions for this code as SPIRV-Cross uses them to communicate errors when parsing SPIRV or compiling SPIRV to Metal. The alternative is that it asserts and terminates the process.

Reply:
We usually try to patch out exceptions from the third-party code instead, but it's probably ok in this case (it's always possible to do it later, if we want to), disabling exceptions is only important for the Web.

Reply:
Sounds good – fortunately, we don't need SPIRV-Cross for Web so hopefully that is ok for the short term. I can look at how complex it might be to remove exceptions in a subsequent PR?